core/src/main/java/com/amplifyframework/predictions/models/SpeechType.java

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.predictions.models;
+
+import androidx.annotation.NonNull;
+
+/**
+ * Different types of parts of speech.
+ *
+ * @see <a href=https://docs.aws.amazon.com/comprehend/latest/dg/how-syntax.html>Analyze Syntax</a>
+ */
+public enum SpeechType {
```

Comment:
Ah that was my original idea too, but recently added this too because it seemed generic enough. But yea I can simply remove the token from enum completely.